api/server/router/local/image.go

```diff
@@ -331,6 +333,11 @@ func (s *router) postBuild(ctx context.Context, w http.ResponseWriter, r *http.R
 	defer output.Close()
 	sf := streamformatter.NewJSONStreamFormatter()
 	errf := func(err error) error {
+		if !buildConfig.Verbose {
+			if notVerboseBuffer.Len() > 0 {
```

Comment:
If you push an update, it might be good to combine these 2 if's into one - but no biggie either way.